src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final double MAX_SIZE = 10.0;
+    private static final String DEFAULT_COLOR = "білий";
```

Comment:
Use only English in messages/code: The value for `DEFAULT_COLOR` should be in English to make the code universal and consistent.

Reply:
fix this